common/common-rest/src/main/java/io/servicecomb/common/rest/codec/param/BodyProcessorCreator.java

```diff
@@ -58,7 +58,8 @@ public Object getValue(RestServerRequest request) throws Exception {
 
                 String contentType = request.getContentType();
                 if (contentType != null && contentType.startsWith(MediaType.TEXT_PLAIN)) {
```

Comment:
I add a TODO to fix this later. But we use a hard code now. Now UTF-8 is the correct one, because most URI encoding use UTF-8 and we use json as body, they are all UTF-8